tests/test_models/test_mutables/test_units/test_dcff_channel_unit.py

```diff
@@ -0,0 +1,14 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+from unittest import TestCase
+
+from mmrazor.models.mutables import DCFFChannelUnit
+
+
+class TestDCFFChannelUnit(TestCase):
+
+    def test_init(self):
```

Comment:
Add more unit cases for DCFFChannelUnit.

Reply:
Updated.